tests/test_manylinux.py

```diff
@@ -225,6 +226,63 @@ def test_build_wheel_with_binary_executable(docker_container):
         python_id, ['python', '-c', 'from testpackage import runit; print(runit(1.5))']).strip()
     assert output.strip() == '2.25'
 
+@pytest.mark.parametrize('with_dependency', ['0', '1'])
+def test_build_wheel_with_highest_policy_and_image_dependencies(with_dependency, docker_container):
+    # try to repair the wheel targeting different policies
+    #
+    # with_dependency == 0
+    #   The python module has no dependencies that should be grafted-in and
+    #   uses versioned symbols not available on policies pre-dating the policy matching the image being tested.
+    # with_dependency == 1
+    #   The python module itself does not use versioned symbols but has a dependency that should be grafted-in that
+    #   uses versioned symbols not available on policies pre-dating the policy matching the image being tested.
+
+    policy, manylinux_id, python_id, io_folder = docker_container
+
+    docker_exec(manylinux_id, ['bash', '-c', 'cd /auditwheel_src/tests/testdependencies && WITH_DEPENDENCY={} python setup.py -v build_ext -f bdist_wheel -d /io'.format(with_dependency)])
```

Comment:
Wouldn't `pip wheel` work here? It would be simpler. There is an example in `test_build_wheel_with_binary_executable` for this.

Reply:
I can't use `pip wheel` because I need to force rebuilding the extension between tests (otherwise the extension used in a previous test will be reused without the correct `WITH_DEPENDENCY` option)